### DISCUSSION TITLE
custom fonts from .ttf files

### DISCUSSION THREAD

**Discussion by ssttaarrllaa** [jgraph/drawio#4073] [Ideas]
Hey! I love draw.io. I use it a lot to make diagrams for work, and stickers for personal use. I use a lot of custom fonts when making stickers. I dont know if its a bug, but very rarely can i get "non-system" fonts to work on draw.io even after I've installed them on my device. Is there anyway that I can just import the .ttf file?

**Comment by OverFloyd** [comment]
You can add custom fonts in the "Text" tab, in the font family menu there's a "custom" button that lets you use all the fonts you want by selecting system ones, from Google Fonts or by just importing the url from somewhere.

**Reply by ssttaarrllaa** [reply]
Thanks! I appreciate the comment. The custom button usually doesnt work for me. Rarely can i get "non-system" fonts to work on draw.io even after I've installed them on my device and then trying to use the custom button. It usually just shows up as the default font in draw.io even if I have a custom font selected. Is there anyway that I can just import the .ttf file?

**Reply by OverFloyd** [reply]
Don't think so, I guess what I said is all the editor offers. I've encountered this issue myself too by the way, but only on some fonts (es. OCR A Extended, which is a base font in Windows) for some reasons. Might open an issue about this, idk what causes them to fail loading.

**Reply by ssttaarrllaa** [reply]
How do I open an issue about this? I saw that this thread had a lot of things like this and they were fixed in time
